README.md

```diff
@@ -42,6 +42,8 @@ module.exports = baseConfig();
 - **Coverage thresholds:** For a good balance  between strict but workable thresholds.
 - **Snapshot serializing:** To remove absolute paths from your snapshots, reducing conflicts in CI.
 
+ℹ️ Test files ending in .spec.js are no longer detected.
```

Comment:
We might want to change this message to something like:

`ℹ️ Test files must be named `test.js` or end with `.test.js` , even if they are inside `__tests__` or any other folder.

Reply:
Done! ✅